perun-rpc/src/main/java/cz/metacentrum/perun/rpc/methods/FacilitiesManagerMethod.java

```diff
@@ -42,6 +42,20 @@ public Facility call(ApiCaller ac, Deserializer parms) throws PerunException {
 		}
 	},
 
+	/*#
+	 * Searches for the Facilities with specified IDs.
```

Comment:
I would use unified javadoc message in RPC for all methods as I suggested for impl layer.

Reply:
Done